src/Connection.cpp

```diff
@@ -6,6 +6,9 @@
 #include "murmur_pch.h"
 
 #ifdef Q_OS_WIN
+#	ifdef _INC_QOS2
```

Comment:
But it still didn't solve the error... there's another check in the header:
`#if (_WIN32_WINNT >= 0x0600)`